src/System-Support/VirtualMachine.class.st

```diff
@@ -372,6 +372,7 @@ VirtualMachine >> isRunningCogit [
 	"Returns true if we're running on the Cog JIT
 	 (vmParameterAt: 46 is the size of the machine code zone)"
 
+	JS ifConnected: [ ^false ].
```

Comment:
Why JS VM could not setup correct value into parameter 46?

Reply:
Currently, the JS VM (when running a non-Cog image) doesn't have that many VM parameters. This can be faked, sure.